### PR TITLE
Add CI, opam and README.md generated from templates in coq-community/manifesto.

### PR DESCRIPTION
**Pull request by Zimmi48**
This PR introduces a new file, `meta.yml`. Together with the templates that can be found in https://github.com/coq-community/manifesto/tree/master/templates, it was used to generate the `.travis.yml`, `default.nix`, `opam` and `README.md` files in this PR, using the command:

```shell
cd /path/to/templates
dir=/path/to/atbr
for f in *.mustache .*.mustache; do
  mustache $dir/meta.yml $f > $dir/${f%.mustache} && \
  echo "$dir/${f%.mustache}"
done
```

The `documentation` content comes from the current version of the `README` and the `description` content comes from the `description` file (that this PR removes).

Note the second commit that is not meant to be merged (or if it is merged it should be reverted later on) which pins the tested version to an older commit of Coq master. Updating to current master requires converting the `.ml4` file into an `.mlg` file and allows dropping the dependency on camlp5. It is also generally advised to split the business logic into another `.ml` file and to keep only the parsing logic in `.mlg`.

### CODE REVIEW

**Comment by Zimmi48** [comment]
Travis does not properly detect newly transferred repositories. I have now manually activated the repo, which should allow to have a build run after closing / reopening.

File: opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+maintainer: "&lt;nil&gt;"
```
Comment (palmskog) [inline opam]: @Zimmi48 can we define some default email for Coq-community OPAM script maintenance (`maintainer` field)? Or should it be the package maintainer (who may not want to make their email public)? In any case, it's clear that this field should not be HTML processed.

Comment (Zimmi48) [reply opam]: Indeed, well spotted. I don't know the package format so well: does it have to be an e-mail and do we have to provide this field? Would you be willing to be the "default" maintainer or should we rather create a coq-community-maintainers mailing list?

Comment (palmskog) [reply opam]: According to the OPAM spec, the maintainer field is required. However, as far as I understand it, there is nothing that forces the maintainer field to be an e-mail address, just a convention used in most OPAM repositories. I can be the default maintainer for now.

**Comment by Zimmi48** [comment]
Updated with the templates from coq-community/manifesto#31.

**Comment by tchajed** [comment]
Thanks @Zimmi48! CI is failing, but it's better than not having CI at all. I merged your pin to an older version so at least something works; once the code is updated I'll update to the latest Coq master. I submitted #4 to track fixing the plugin.

**Pull request merged**